src/unk_0202D938.c

```diff
@@ -0,0 +1,26 @@
+#include "save.h"
+
+u32 sub_0202D938(void) {
+    return 0xe;
+}
+
+void sub_0202D93C(u8* argP) {
```

Comment:
This function takes in a struct*, not a u8*. This struct should be placed in a header file, along with declarations for each of the three functions here.

Something like this should be fine for now:
```c
typedef struct UnkSavStruct20 {
    u8 filler_00[14];
} UnkSavStruct20;
```

Reply:
How can i tell when a function is taking in a struct in the future?